### DISCUSSION TITLE
Convite para fazer parte da Angola Open-source Community Mentorship

### DISCUSSION THREAD

**Discussion by antonio-pedro99** [RocketChat/Rocket.Chat#29962] [General]
Olá, equipe [RocketChat](https://github.com/RocketChat)

Espero que estejam bem. Sou Antonio Pedro da [Angola Open-source Community](https://github.com/angolaosc), uma comunidade de desenvolvedores dedicados a promover o desenvolvimento open-source em Angola.

Estamos atualmente organizando um Programa de Mentoria em Desenvolvimento Open-source em Angola, visando capacitar os nossos desenvolvedores locais e estimular a colaboração com projetos de código aberto de renome internacional.

Gostaríamos de convidar o [RocketChat](https://github.com/RocketChat) para participar como uma organização mentora no nosso programa. Acreditamos que a participação deste projecto como mentor pode ser extremamente valiosa para nossos desenvolvedores, permitindo que eles contribuam ativamente para um projeto relevante e aprendam com as melhores práticas de uma organização experiente.

O nosso [programa](https://github.com/angolaosc/mentorship) terá duração de um mês e contará com diferentes etapas, incluindo pré-mentoria(onde nós a organização, iremos conduzir um workshop introdutório sobre open-source), mentoria, onde os desenvolvedores terão a oportunidade de trabalhar em qualquer issue aberta neste projeto, receber feedback de especialistas e desenvolver as suas habilidades em desenvolvimento de software.

Seria uma honra contar com o [RocketChat](https://github.com/RocketChat) como um mentor no nosso programa. Acreditamos que essa colaboração pode trazer benefícios mútuos, fortalecendo a comunidade open-source em Angola e promovendo a inovação tecnológica em ambos os lados.

Se estiverem interessados em participar ou tiverem-se alguma dúvida, por favor, deixem-nos saber. Estamos ansiosos para uma possível parceria com o [[RocketChat](https://github.com/RocketChat)

Agradecemos a sua atenção e consideração.

Atenciosamente,
Antonio Pedro
Angola Open-source Community

**Comment by debdutdeb** [comment]
Hi @antonio-pedro99 , sorry for the delay in noticing this post. Could you allow us one more week to discuss this internally, please?

**Reply by antonio-pedro99** [reply]
Hi @debdutdeb, thanks for your feedback. 
Please take your time
